internal/platform/platform.go

```diff
@@ -72,6 +75,13 @@ func detect() Platform {
 	p := Platform{
 		HostInfo: info,
 	}
+	if resourcedetector.OnBMS() {
+		p.ResourceOverride, err = resourcedetector.GetResource()
```

Comment:
Why aren't you calling `resourcedetector.GetBMSResource()`?

(Also, beware typed vs untyped `nil`)

Reply:
I accidentally mis-typed the method name. Fixed.